src/index.ts

```diff
@@ -17,11 +17,23 @@ const client = new Client({
 const channelIDs: string[] = process.env.CHANNELIDS?.split(',') || [];
 const trigger: string = process.env.TRIGGER || '!llama';
 
+// Add the textChannel 
+let textChannel: TextChannel;
+
 client.once('ready', () => {
-    console.log('Ollama Online!');
+
+  // Get the text channel by ID
+  textChannel = client.channels.cache.get(channelIDs[0]) as TextChannel;
+
+  // Send startup messages
+  textChannel.send('Ollama Online! Ready to receive messages preceded by ' + trigger);
```

Comment:
I'm thinking we remove this, i know for my server i'd rather not have the message pop up every time the bot starts up

Perhaps just comment it out so people can easily adjust if they so wish

Reply:
Great suggestion. I've done so and updated the fork.